backend/tracim_backend/lib/core/content.py

```diff
@@ -419,26 +419,27 @@ def get_base_query(
     # 
     #     return result
 
-    def _is_content_label_free(
+    def _is_filename_free(
```

Comment:
what do you think of renaming this method to `_is_filename_available` which is (on my point of view) more exact for the case.